tools/docker/build_wheel.Dockerfile

```diff
@@ -72,6 +51,10 @@ ENV TF_CUDA_VERSION=$TF_CUDA_VERSION
 ENV TF_CUDNN_VERSION=$TF_CUDNN_VERSION
 
 RUN python -m pip install --upgrade pip
+
+# Avoid SystemError: initialization of _pywrap_checkpoint_reader raised unreported exception
+RUN python -m pip install tensorflow==$TF_VERSION
```

Comment:
Yeah, for supporting multi-version Tensorflow, the CI docker images would not install TF until CI starts because the image size should not be too large.